service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/GetAbsentAttachmentTaskExecutor.java

```diff
@@ -33,11 +36,11 @@ public Class<GetAbsentAttachmentTaskDefinition> getTaskType() {
 
     @Override
     public void execute(GetAbsentAttachmentTaskDefinition taskDefinition) {
-        var ehrExtractStatus = handleAbsentAttachment(taskDefinition);
+        var ehrExtractStatus = handleAbsentAttachment(taskDefinition, Optional.empty());
         detectTranslationCompleteService.beginSendingCompleteExtract(ehrExtractStatus);
     }
 
-    public EhrExtractStatus handleAbsentAttachment(DocumentTaskDefinition taskDefinition) {
+    public EhrExtractStatus handleAbsentAttachment(DocumentTaskDefinition taskDefinition, Optional<String> exceptionDisplay) {
```

Comment:
`exceptionDisplay` is a little confusing a name. I think `display` is quite low level implementation detail of where the value comes from, as opposed to what it represents.

I'm thinking `responseErrorMessage` or something which makes it clear that this is an error which has come from making a request for the document, and we got an error back. (I'm not super happy with `responseErrorMessage`)

Reply:
have renamed this to gpcResponseError as this seemed clearest to me